src/components/AddItem/AddItem.component.jsx

```diff
@@ -1,7 +1,5 @@
 import React, { useState } from 'react';
-// import * as firebase from '../Firebase/Firebase.component';
-import 'firebase/firestore';
-import * as firebase from 'firebase/app';
```

Comment:
Thank you for updating. Consist naming convention and utilization of existing code base is important to remember.